source/guides/dashboard/runs.md

```diff
@@ -22,6 +22,40 @@ Details of each run are displayed including:
 
 {% imgTag /img/dashboard/run-details.png "run-details" %}
 
+# Cancelling runs
+
+You can cancel a run currently in progress from the Dashboard. Runs can only be cancelled by members of a project.
+
+To cancel a run, click on a run in progress from the run list, then click on "Cancel Run" in the upper-right corner of the run details page.
+
+Cancelling a run in progress cannot be undone. Once you click "Cancel Run", a confirmation message will appear. Click the red "Yes, cancel this run" button to proceed.
+
+{% video local /img/snippets/cancelling-run.mp4 %}
+
+Once a run is cancelled:
+
+- The run status will update to cancelled
+- All incompleted instances of the run will be marked as cancelled
+- All test runners associated with the run will exit with an error
```

Comment:
I don't understand this point. I think this is meant to mean that any `cypress run` currently in progress associated with the run will exit with an error code, but I'm not sure this is actually how this behaves.

Reply:
That is correct, it does throw an error. I can add a screenshot of what that error output looks like.